ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Paged/RequestBlockingScript.js

```diff
@@ -9,10 +9,12 @@ window.__firefox__.execute(function($) {
   const messageHandler = '$<message_handler>';
   const blockingCache = new Map();
   const sendMessage = $((resourceURL) => {
+    // when location.href does not match origin, we include origin in console
+    const originDisplay = window.location.href !== window.origin ? ` (${window.origin})` : ``;
```

Comment:
Why do you need `(${window.origin})`? You can just use `window.origin` no? Likewise do we need empty back-ticks instead of `''`?

Reply:
This is just in the log to try and be a bit more helpful / descriptive when the request comes from an `about:blank` iframe it will report as `Brave prevented frame displaying about:blank (brave.com) from loading a resource from brave.com/some-resource.js` (if we only displayed `window.origin` it wouldn't be clear it came from the iframe as only `brave.com` would be listed). 

I can replace the empty backticks with `''`.

Reply:
Just to note that there will be many cases where `window.location.href !== window.origin` that are irrelevant to local frames.  `window.location.href` is the full URL of the current frame, while `window.origin` is the security origin. For this github page, the former is `https://github.com/brave/brave-core/pull/26622`, and the latter is `https://github.com/`.

If this check is intended to just check "is this an about:blank" frame, you might want something like `window.location.origin !== window.origin`

Reply:
Yeah the intention was to provide more info for all cases - not just `about:blank` - as the request(s) are now blocked against `window.origin` not `window.location.href`. I can just log `window.origin` here and leave out the `window.location.href` from the console info log if this is causing confusion, the intention was just to be a bit more descriptive (otherwise the log wouldn't indicate if the request was blocked on the main frame or from an iframe).